docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.batch_size                  | integer         | Dev set batch size                                                                                           |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.name                                | string          | Optimizer to use for training                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.lr                                  | float           | Learning rate to use for training                                                                            |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.weight_decay                        | float           | Weight decay to use for training                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.name                          | string          | Warm up schedule                                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.warmup_ratio                  | float           | Warm up ratio                                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+To train the model from scratch, run:
+
+.. code::
+
+      python examples/nlp/token_classification/punctuation_and_capitalization_train.py \
+             model.dataset.data_dir=<PATH/TO/DATA_DIR> \
+             trainer.gpus=[0,1] \
+             optim.name=adam \
+             optim.lr=0.0001 \
+             model.nemo_path=<PATH/TO/SAVE/.nemo>
+
+To train from the pre-trained model, use:
+
+.. code::
+
+      python examples/nlp/token_classification/punctuation_and_capitalization_train.py \
+             model.dataset.data_dir=<PATH/TO/DATA_DIR> \
+             pretrained_model=<PATH/TO/SAVE/.nemo>
+
+
+Required Arguments for Training
```

Comment:
Having optional arguments does not make sense as everything other than required are optional and we are not limited and simplified like TLT. I feel kie we can drop the optional argument part. You may have just one section: arguments and put all descriptions there. You may mention this one is required. Otherwise you need to explain all arguments in the optional section, not just some selected ones.